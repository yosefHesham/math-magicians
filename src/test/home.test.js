import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/home';

describe('Test for home page', () => {
  test('Quote page should render correctly', () => {
    const home = renderer
      .create(<Home />).toJSON();

    expect(home).toMatchSnapshot();
  });
});
