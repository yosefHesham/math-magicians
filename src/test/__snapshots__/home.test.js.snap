// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for home page Quote page should render correctly 1`] = `
<div
  className="home-wrapper"
>
  <h2>
     Welcome to our page !
  </h2>
  <p
    className="home-content"
  >
    LoremIpsumLoremIpsumLoremIpsumLoremIpsumLoremI psumLoremIpsumLoremIpsumLoremIpsumLoremIpsu mLoremIpsumLoremIpsumLoremIpsumLoremIpsumLoremIp sumLoremIpsumLoremIpsumLoremIpsumLoremIpsumLoremIpsumL oremIpsumLoremIpsumLoremIpsumLoremIpsumLoremIpsumLoremIpsum
  </p>
</div>
`;
